.eslintrc.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+const tsConfigurations = [
+  'eslint:recommended',
+  'plugin:@typescript-eslint/recommended',
+  'plugin:@typescript-eslint/recommended-requiring-type-checking',
```

Comment:
I just decided to adapt the Compass eslint config here, yeah. It does make for a really nice DX imo, especially the prettier integration that VSCode can pick up on (I assume that’s happening because it’s part of the eslint config)

I’ve started out using `eslint-config-compass` but that has a few dependencies that it doesn’t list explicitly (the ones that are hoisted to the top-level package.json in the Compass monorepo) and also a bunch of things we don’t need like react integrations, so this file is really just a trimmed-down version of it.

Reply:
I’ll merge and go ahead with this for now, but happy to adjust